src/js/base/module/Editor.js

```diff
@@ -744,8 +744,9 @@ export default class Editor {
   fontStyling(target, value) {
     const rng = this.getLastRange();
 
-    if (rng) {
+    if (rng != '') {
```

Comment:
`rng` can be null or valid range object. So this comparison should be reverted as it was.